f2/src/forms/LocationInfoForm.js

```diff
@@ -55,24 +56,23 @@ export const LocationInfoForm = (props) => {
               />
             )}
             <Container>
-              <Row>
+              <FormRow>
                 <Trans id="locationinfoPage.skipInfo" />
-              </Row>
-              <Row>
+              </FormRow>
+              <FormRow marginBottom="2rem">
```

Comment:
I compared marginBottom="2rem" and marginBottom="1rem". feel 1rem looks better.

Reply:
Yes, 1 rem is consistent with Contact Info form.